helm/charts/determined/templates/master-config.yaml

```diff
@@ -56,6 +56,9 @@ stringData:
       {{- end }}
 
     security:
+      {{- if .Values.initialUserPassword }}
+      initial_user_password: {{ .Values.initialUserPassword | quote }}
```

Comment:
omg, just rebased to this and -- thanks!! Saves me some work on [DET-10208](https://hpe-aiatscale.atlassian.net/browse/DET-10208) I think!